lib/whitehall_importer/create_revision.rb

```diff
@@ -16,9 +16,10 @@ def self.call(*args)
       new(*args).call
     end
 
-    def initialize(document_import, whitehall_edition)
+    def initialize(document_import, whitehall_edition, change_history = [])
```

Comment:
Do we have any advantages of this defaulting to an empty array?

Reply:
The advantage would be not having to add extra code to the create_revision_spec, think theres around 25 tests so seemed worthwhile, what do you think?

Reply:
Yup sounds a good reason 👍